dts/bindings/cpu/nordic,riscv.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nordic Semiconductor RISC-V CPU
+
+compatible: "nordic,riscv"
```

Comment:
nit: should be `nordic,vpr`

Reply:
Or `nordic,nrf-vpr`?

Reply:
> Or `nordic,nrf-vpr`?

yup

Reply:
I'd actually stick to `nordic,vpr`, since it's a RISC-V core that could be used out of nRF products, who knows.